101-databricks-workspace-with-customer-managed-keys/azuredeploy.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "workspaceName": {
```

Comment:
remove all parameters for which there are defaultValues

Reply:
Why do we want to remove values in parameters file and use default values? Is there any reason ??